optimum/onnxruntime/trainer.py

```diff
@@ -459,6 +459,11 @@ def _inner_training_loop(
 
         # Wrap the model with `ORTModule`
         logger.info("Wrap ORTModule for ONNX Runtime training.")
+
+        from onnxruntime import version as ort_version
+        ort_support_stage3 = version.parse(ort_version) >= version.parse("1.17.0")
+        os.environ["ORTMODULE_ENABLE_ZERO_STAGE3"] = str(int(ort_support_stage3 and self.is_deepspeed_enabled and is_deepspeed_zero3_enabled()))
```

Comment:
Is it possible to override the runtime options after wrapping the model with `ORTModule`?  If so, it might be better to put it under the deepspeed-releated code below: https://github.com/huggingface/optimum/blob/15b8d1eed4d83c5004d3b60f6b6f13744b358f01/optimum/onnxruntime/trainer.py#L471-L472

Reply:
unfortunately, the env var is pick up during the wrap, so need to set before ORTModule instantiated.

Reply:
Got it, thanks for the info!